static/templates/user_sidebar_actions.handlebars

```diff
@@ -21,7 +21,7 @@
   <li>
     <a class="star_user">
       <i class="icon-vector-star"></i>
-      {{#tr this}}{{#if starred}}{{t Unstar}}{{else}}{{t Star}}{{/if}} {{t user}} <b>__name__</b>{{/tr}}
+      {{#tr this}}{{#if starred}}{{t "Unstar"}}{{else}}{{t "Star"}}{{/if}} {{t user}} <b>__name__</b>{{/tr}}
```

Comment:
You can write this line like this as well:
`{{#tr this}}{{#if starred}}"Unstar"{{else}}"Star"{{/if}} user <b>__name__</b>{{/tr}}`